pkg/provisioner/ironic/ironic.go

```diff
@@ -1239,6 +1239,59 @@ func (p *ironicProvisioner) Prepare(data provisioner.PrepareData, unprepared boo
 	return
 }
 
+func (p *ironicProvisioner) getConfigDrive(data provisioner.ProvisionData) (configDrive nodes.ConfigDrive, err error) {
+	// In theory, Ironic can support configdrive with live ISO by attaching
+	// it to another virtual media slot. However, Dell does not support two
+	// virtual media devices at the same time, so we shouldn't try it.
```

Comment:
From recent discussions with @rhjanders I don't think Supermicro does either, so we should perhaps just say this is not reliably/commonly supported on all hardware types?